src/Wallabag/CoreBundle/Twig/WallabagPiwikExtension.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Wallabag\CoreBundle\Twig;
+
+use Craue\ConfigBundle\Util\Config;
+use PiwikTwigExtension\PiwikTwigExtension;
+
+class WallabagPiwikExtension extends PiwikTwigExtension
+{
+    public function __construct(Config $craueConfig)
+    {
+        parent::__construct($craueConfig->get('piwik_host'), $craueConfig->get('piwik_site_id'), $craueConfig->get('piwik_enabled'));
```

Comment:
Could you split them line by line? I'll more readable.
Also, could you add a comment on the class telling why you are doing that?